hearth/lib/hearth/stubs.rb

```diff
@@ -3,14 +3,13 @@
 module Hearth
   # Provides a thread safe data structure for adding and getting stubs
   # per operation.
-  # @api private
   class Stubs
```

Comment:
Do we want a private Stub class to wrap the stubs? That way our `send` middleware is less exhaustive with setting stubs and possibly with testing too? The Stub class could probably validate that stubs are a proc, or Response, or hash, or error, etc?

Reply:
Yeah, a stub class could potentially do validation here, but lets consider in a separate PR since this is already a pretty big change.